interactive/update_environment_variables.py

```diff
@@ -0,0 +1,25 @@
+import os
+import logging
+import json
+
+# open settings.json and set environment variables
+def update_environment_variables():
+    __location__ = os.path.realpath(
+    os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+    settings_location = os.path.join(__location__, 'settings.json')
+    with open(settings_location) as f:
+        settings = json.load(f)
+        os.environ["rabbit-mq-host"] = settings["rabbit-mq-host"]
```

Comment:
The lines assigning into `os.environ` can be replaced with `os.environ.update(settings)`?